src/detect-flow-age.c

```diff
@@ -29,10 +29,13 @@ static int DetectFlowAgeMatch(
     if (p->flow == NULL) {
         return 0;
     }
-    uint32_t age = SCTIME_SECS(p->flow->lastts) - SCTIME_SECS(p->flow->startts);
+    uint64_t age = SCTIME_SECS(p->flow->lastts) - SCTIME_SECS(p->flow->startts);
+    if (age > UINT32_MAX) {
+        age = UINT32_MAX;
+    }
```

Comment:
Should we change flow.age behavior for a u64 ? cc @inashivb

Reply:
think we'll have many flows that will be longer than a 100 years?

Reply:
I can craft a pcap file like that but should I...